src/jdk.jdeps/share/classes/com/sun/tools/classfile/Attribute.java

```diff
@@ -48,6 +48,7 @@
     public static final String EnclosingMethod          = "EnclosingMethod";
     public static final String Exceptions               = "Exceptions";
     public static final String InnerClasses             = "InnerClasses";
+    public static final String JavaFlags                = "JavaFlags";
     public static final String LineNumberTable          = "LineNumberTable";
```

Comment:
The name and attribute syntax was suggested by Dan, this may incur/undergo some bikeshedding later.